UQ-eLIPSE/crucible-components
# Story/VETHUB-19

Selecting question options

This is mostly styling but now I'm using variable state managing by using `ref` (You used to use the data property but Vue 3 with their composition API have changed that).

Essentially added styling for 3-4 classes and changes the state when option is selected.

Next ticket will deal with correctness.

To test:

-     Run yarn dev and then navigate to https://localhost:5173
-     Click the option and it should remain selected.
-     Change options so that it transitions smoothly.
-     Run yarn test to run the updated tests.
